From 38e5723c4d935061006c555ff701eeb618bb2945 Mon Sep 17 00:00:00 2001
From: Gareth Williams <gareth.williams.jx@renesas.com>
Date: Tue, 15 Sep 2020 14:06:42 +0100
Subject: [PATCH] Use wget instead of curl to fetch https source

This is a workaround to avoid the error:
"curl: (77) error setting certificate verify locations"

Signed-off-by: Gareth Williams <gareth.williams.jx@renesas.com>
---
 tensorflow/lite/tools/make/download_dependencies.sh | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/tensorflow/lite/tools/make/download_dependencies.sh b/tensorflow/lite/tools/make/download_dependencies.sh
index 27537823be..b549411047 100755
--- a/tensorflow/lite/tools/make/download_dependencies.sh
+++ b/tensorflow/lite/tools/make/download_dependencies.sh
@@ -77,7 +77,8 @@ download_and_extract() {
   rm -rf ${dir}/*  # Delete existing files.
   tempdir=$(mktemp -d)
   filepath="${tempdir}/$(basename ${url})"
-  curl -Lo ${filepath} ${url}
+  wget -qO ${filepath} "${url}"
+
   if [ -n "${sha256}" ]; then
     echo "checking sha256 of ${dir}"
     echo "${sha256}  ${filepath}" | sha256sum -c
-- 
2.25.1

